### DISCUSSION TITLE
XCH has broken $200 HPOOL shuts down hpool.com - The end is near

### DISCUSSION THREAD

**Discussion by room101-dev** [Chia-Network/chia-blockchain#6888] [Ideas]
Well its been a bloody night here in ASIA, all the plugs are now pulled on Chia, the official word is out "Green" means nothing, the CCP is taking it all down, if it crypto, its time to CYA.

1,000 plots now earns less than $2/day, that's $5k USD worth of HDD's making a whopping $60/month, now the bad, every few days you get 1/2 less, and less. Price going to zero, and space going to 100k EB; Soon 1,000 plots will earn a nickel a day.

Hpool.com might come back as another name, but its clear, they're moving their domain to be nearer and/or closer to mother chia-net in the USA.

**Comment by kanasite** [comment]
https://hpool.co/
